bot/controller.py

```diff
@@ -37,3 +41,18 @@ def create_chatbot(self, request: BotCreate):
             raise HTTPException(status_code=400, detail="Unsupported message adapter")
         except Exception:
             raise HTTPException(status_code=500, detail="Something went wrong")
+        
+    def update_chatbot(self, bot_id, request: BotUpdate):

```

Comment:
I think the parameters of this concrete method should be adjusted to match those of the abstract method.

Reply:
great catch, ty